### PR TITLE
[doc] Add 3.0-version migration docs

### PR DESCRIPTION
**Pull request by kobawan**
### Description

To help migrating from version 2 to version 3.

### Type
- [ ] Breaking change
- [ ] Enhancement
- [ ] Fix
- [x] Documentation
- [ ] Tooling

### CODE REVIEW

File: docs/api/3.0-migration.md [hunk 1]
```diff
@@ -215,3 +215,39 @@ Due to a bug where `originalUrl` was being assigned not as the root url, but the
 * Parameter `originalUrl` from `resolveWrappers` has been correctly renamed to `previousUrl`: `resolveWrappers(ad, wrapperDepth, previousUrl)`
 * `resolveAds` param `originalUrl` in `options` object has been renamed to `previousUrl`. It also can be passed `url` into `options`. For example: `resolveAds(ads = [], { url, previousUrl })`
 * `options` parameter in `VASTParser` `parse` function now expects `previousUrl` instead of `originalUrl`. It also can be passed `url` into `options`.
+
+#### Improvement for vast parser to trigger warnings event logs
```
Comment (kobawan) [inline docs/api/3.0-migration.md]: this isn't actually a breaking change, because it is in a util file not exported in the module

**Comment by jakub-g** [comment]
The fixes for Windows in #340 require node 10.12+ so perhaps you could add a word that min version of node is changed.

File: docs/api/3.0-migration.md [hunk 2]
```diff
@@ -0,0 +1,306 @@
...
+We now require node version 10.12+
+
+## Version 3.0.0
+
+### Sequence attribute value overriden by wrapper parent sequence value
+
+The sequence attribute of an Inline ad when wrapped in wrapper will be replaced by the one of its parent wrapper.
+The replacement will be made only if the parent wrapper contains only one ad. Otherwise sequences from the unwrapped ad will be kept.
+
+e.g,for a use case where an AdPod contain:
```
Comment (clementFrancon) [inline docs/api/3.0-migration.md:283]:
```suggestion
e.g, for a use case where an AdPod contain:
```

File: docs/api/3.0-migration.md [hunk 3]
```diff
@@ -0,0 +1,306 @@
...
+
+### NodeJS EventEmitter replaced with custom class
+
+A lighter custom class with the same syntax has been created to replace nodeJS EventEmitter. API docs for custom class described [here](https://github.com/dailymotion/vast-client-js/blob/3.0-version/docs/api/event-emitter.md).
+
+## Version 3.0.0-alpha.1
+
+### No more imports from `src`
+
+`module` package.json entry with pre-build files has been replaced with `browser` entry pointing towards already built files. This is to facilitate packages still using babel 6 (as it is not compatible with babel 7).Check [README](https://github.com/dailymotion/vast-client-js/blob/3.0-version/README.md#api-documentation) file for more detailed info.
```
Comment (clementFrancon) [inline docs/api/3.0-migration.md:159]:
```suggestion
`module` package.json entry with pre-build files has been replaced with `browser` entry pointing towards already built files. This is to facilitate packages still using babel 6 (as it is not compatible with babel 7). Check [README](https://github.com/dailymotion/vast-client-js/blob/3.0-version/README.md#api-documentation) file for more detailed info.
```

**Review by clementFrancon — APPROVED** [review]
Just a small missing spaces otherwise, good job 💯

**Pull request merged**